Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -865,6 +865,29 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
         }
     }
 
+    /// Add users to the channel as members.
+    ///
+    /// - Parameters:
+    ///   - userIds: User ids that will be added to a channel.
+    ///   - hideHistory: Hide the history of the channel to the added member. By default, it is false.
+    ///   - message: Optional system message sent when adding members.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
+    ///                 If request fails, the completion will be called with an error.
+    @available(*, deprecated, message: "A new addMembers function is now available that supports adding MemberInfo instead of only the user id.")
```

Comment:
About the deprecation - do we want to remove the old methods in a major release? I think we can keep them as an alternative / simpler version.

Reply:
Yeah I thought about this as well. Since you mentioned this, it means it probably makes sense, so I will rever this 👍

Reply:
Done ✅